reactor-netty-http/src/test/java/reactor/netty/http/client/WebsocketClientOperationsTest.java

```diff
@@ -67,7 +68,7 @@ private void failOnClientServerError(
 				          .port(httpServer.port())
 				          .wiretap(true)
 				          .headersWhen(h -> login(httpServer.port()).map(token -> h.set("Authorization", token)))
-				          .websocket(WebsocketClientSpec.builder().protocols(clientSubprotocol).build())
+				          .websocket(WebsocketClientSpec.builder().version(WebSocketVersion.V08).protocols(clientSubprotocol).build())
```

Comment:
Instead of adding here a configuration for the version, I suggest to have a dedicated test for this feature.
Wdyt?

Reply:
Better idea indeed, I'll be able to test failing cases this way